dataPipelines/gc_downloader/doc_utils.py

```diff
@@ -14,7 +14,7 @@
 
 def get_pdf_downloadable_item(doc: Document) -> Optional[DownloadableItem]:
     """Get pdf downloadable item corresponding to doc"""
-    return next((i for i in doc.downloadable_items if i.doc_type.lower() == 'pdf'), None)
+    return next((i for i in doc.downloadable_items if i.doc_type.lower() == 'pdf' or i.doc_type.lower() == 'html'), None)
```

Comment:
should we separate this into different functions for clarity? get_pdf_downloadable_item probably shouldn't return htmls as well

Reply:
I think we should rename this function to "get_supported_downloadable_items" and check `if i.doc_type.lower() in SUPPORTED_FILE_EXTENSIONS` instead of checking individually